### PR TITLE
Update Datasets Blog Post

### PR DESCRIPTION
**Pull request by cfahlgren1**
## Notes / Goals
We want to create a blog post that encapsulates all of the “out of the box” features you get when you host a dataset on Hugging Face. We have tons of features for datasets that make it best in class and there are a lot of opportunities to boost our ranking for search related to hosting datasets.

**We want to optimize the content for Perplexity / ChatGPT / Google Instant Answer with our Hub Features on Datasets**

### CODE REVIEW

File: researcher-dataset-sharing.md [hunk 1]
```diff
@@ -3,19 +3,22 @@ title: "Creating open machine learning datasets? Share them on the Hugging Face
...
 
-This post outlines:
+Many of the world's leading research institutions and companies such as Meta, Google, OpenAI, and THUDM are using the Hugging Face Hub to host their datasets. By hosting a dataset on the Hugging Face Hub, you instantly get access to many features that can maximize the impact of your dataset:
 
-- Why researchers should openly share their data (feel free to skip this section if you are already convinced about this!)
-- What the Hugging Face Hub offers for researchers who want to share their datasets.
-- Resources for getting started with sharing your datasets on the Hugging Face Hub.
+- Generous Limits
+- Dataset Viewer
+- Third Party Library Support
```
Comment (cfahlgren1) [inline researcher-dataset-sharing.md:18]: Wonder if we should remove or shorten this section so that we can keep things more concise and specific to the hub?
https://github.com/huggingface/blog/blob/main/researcher-dataset-sharing.md?plain=1#L66-L94

Maybe we can replace it with the *Third Party Library Support* section from [here](https://docs.google.com/document/d/1dv3VsugiQr_2u3I5WvNb4zxPVMZj8kBSmALrDW2otlM/edit#heading=h.aseu0z813b7x) since that may be more enticing to dataset practitioners?

wdyt @davanstrien 

or maybe removing it and updating this https://github.com/huggingface/blog/blob/main/researcher-dataset-sharing.md?plain=1#L49-L53 with Third Party Library Support?

File: researcher-dataset-sharing.md [hunk 2]
```diff
@@ -3,19 +3,22 @@ title: "Creating open machine learning datasets? Share them on the Hugging Face
...
+- SQL Console
+- Access Controls
+- Reach and Visibility
 
 ## Why share your data?
```
Comment (cfahlgren1) [inline researcher-dataset-sharing.md:23]: Maybe we can remove this section and the "What is the Hugging Face Hub" section and just dive into why user should use the hub? If we want to keep it maybe we can reduce this to one sentence?

My hunch is most people skim blog posts and we are aiming for SEO / LLM parseability.

File: researcher-dataset-sharing.md [hunk 3]
```diff
@@ -3,19 +3,22 @@ title: "Creating open machine learning datasets? Share them on the Hugging Face
...
 
-Are you a researcher doing data-intensive research or using machine learning as a research tool? As part of this research, you have likely created datasets for training and evaluating machine learning models, and like many researchers, you may be sharing these datasets via Google Drive, OneDrive, or your own personal server. In this post, we’ll outline why you might want to consider sharing these datasets on the Hugging Face Hub instead. 
+Are you a researcher doing data-intensive research or using machine learning as a research tool? As part of this research, you have likely created datasets for training and evaluating machine learning models, and like many researchers, you may be sharing these datasets via Google Drive, OneDrive, or your own personal server. In this post, we’ll outline why Hugging Face Hub is the best place on the internet to host open datasets.
 
-This post outlines:
+Many of the world's leading research institutions and companies such as Meta, Google, OpenAI, and THUDM are using the Hugging Face Hub to host their datasets. By hosting a dataset on the Hugging Face Hub, you instantly get access to many features that can maximize the impact of your dataset:
```
Comment (davanstrien) [inline researcher-dataset-sharing.md:14]: It would be nice to list some universities here, too.

File: researcher-dataset-sharing.md (hunk 2 above)
Comment (davanstrien) [reply researcher-dataset-sharing.md:23]: sg, happy for you to update the PR with these suggested changes

File: _blog.yml [hunk 4]
```diff
@@ -2974,7 +2974,7 @@
   title: "Creating open machine learning datasets? Share them on the Hugging Face Hub!"
   author: davanstrien
   thumbnail: /blog/assets/researcher-dataset-sharing/thumbnail.png
-  date: October 30, 2023
+  date: October 30, 2024
```
Comment (pcuenca) [inline _blog.yml:2977]: haha nice yearly update!

Let me know if/when you want a review.

File: researcher-dataset-sharing.md (hunk 3 above)
Comment (cfahlgren1) [reply researcher-dataset-sharing.md]: Any good ones that use datasets? I added Stanford, but I am sure there are some others that are more active. Feel free to add a commit suggestion.

File: researcher-dataset-sharing.md [hunk 5]
```diff
@@ -3,134 +3,106 @@ title: "Creating open machine learning datasets? Share them on the Hugging Face
...
-Datasets shared on the Hugging Face Hub can be loaded via a variety of tools. The [`datasets`](https://huggingface.co/docs/datasets/) library is a Python library which can directly load datasets from the huggingface hub via a `load_dataset` command. The `datasets` library is optimized for working with large datasets (including datasets which won't fit into memory) and supporting machine learning workflows. 
+- [Generous Limits](#generous-limits)
+- [Dataset Viewer](#dataset-viewer)
+- [Third Party Library Support](#third-party-library-support)
+- [SQL Console](#sql-console)
+- [Access Controls](#access-controls)
+- [Reach and Visibility](#reach-and-visibility)
 
-Alongside this many of the datasets on the Hub can also be loaded directly into [`Pandas`](https://pandas.pydata.org/), [`Polars`](https://www.pola.rs/), and [`DuckDB`](https://duckdb.org/). This [page](https://huggingface.co/docs/datasets-server/parquet_process) provides a more detailed overview of the different ways you can load datasets from the Hub.
+##  Generous Limits
```
Comment (cfahlgren1) [inline researcher-dataset-sharing.md:26]: Some good things to add to this section:

- Generous limits for free
- Highlight future support and improvements for even bigger datasets via XetHub cc: @erinys

File: researcher-dataset-sharing.md [hunk 6]
```diff
@@ -3,134 +3,106 @@ title: "Creating open machine learning datasets? Share them on the Hugging Face
...
+## SQL Console
 
-<p align="center"> 
- <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/researcher-dataset-sharing/filesize.png" alt="Screenshot of the file size information for a dataset"><br> 
-<em>The Hugging Face Hub can host the large datasets often created for machine learning research.</em> 
- </p> 
-
-
-## API and client library interaction with the Hub
+## Access Controls
```
Comment (cfahlgren1) [inline researcher-dataset-sharing.md:77]: Needs to be filled out. Some things to add / highlight:

- Gating (restricting access)
- Private Repos
- Small Snippet about further security controls etc via Enterprise Plan

File: _blog.yml (hunk 4 above)
Comment (cfahlgren1) [reply _blog.yml:2977]: Will do, still in the add phase and using GitHub to collaborate so feel free to add / fill in any sections.

If not, I can let you know when it's ready for review / revision :)

File: researcher-dataset-sharing.md [hunk 7]
```diff
@@ -1,136 +1,177 @@
 ---
-title: "Creating open machine learning datasets? Share them on the Hugging Face Hub!"
+title: Share your open ML datasets on Hugging Face Hub!"
```
Comment (pcuenca) [inline researcher-dataset-sharing.md:2]:
```suggestion
title: "Share your open ML datasets on Hugging Face Hub!"
```

File: researcher-dataset-sharing.md [hunk 8]
```diff
@@ -1,136 +1,177 @@
...
 thumbnail: /blog/assets/researcher-dataset-sharing/thumbnail.png
```
Comment (pcuenca) [inline researcher-dataset-sharing.md:3]: Maybe you want to refresh as well? (not critical at all)

File: _blog.yml (hunk 4 above)
Comment (pcuenca) [reply _blog.yml:2977]: A couple of details regarding the metadata file:
- We need to make sure the title is in sync with the one in the post.
- If you want the post to appear in the first page of hf.co/blog, we have to move the entry to the end of the yaml file.

**Review by pcuenca — COMMENTED** [review]
Just checked metadata / format for now, will revisit later :)

**Comment by cfahlgren1** [comment]
cc: @lhoestq @severo for the dataset viewer section. Feel free to update / suggest anything that may be valuable there. Added new video support there.

File: sql-console.md [hunk 9]
```diff
@@ -14,7 +14,7 @@ We are very excited to announce that you can now run SQL queries on your dataset
 
 ## Introducing the SQL Console for Datasets
 
-On every public dataset you should see a new **SQL Console** badge. With just one click you can open a SQL Console to query that dataset.
```
Comment (davanstrien) [inline sql-console.md:17]: Should this file be in a different PR?

Comment (cfahlgren1) [reply sql-console.md:17]: Since it’s just deleting one word “public” and is related to this PR (we will link to that post) and use similar language, we probably wouldn’t want conflicting info. wdyt?

File: researcher-dataset-sharing.md [hunk 10]
```diff
@@ -1,143 +1,195 @@
...
+<p align="center"> 
+ <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/researcher-dataset-sharing/filesize.png" alt="Screenshot of the file size information for a dataset"><br> 
+<em>The Hugging Face Hub can host the large datasets often created for machine learning research.</em> 
+ </p> 
 
-Domain-specific datasets are vital for evaluating and training machine learning models, helping to overcome the limitations of existing models. Creating these datasets, however, is challenging, requiring significant time, resources, and domain expertise, particularly for annotating data. Maximizing the impact of this data is crucial for the benefit of both the researchers involved and their respective fields.
+_Note: The [Xet team](https://huggingface.co/xet-team) is currently working on a backend update that will increase per-file limits from the current 50 GB to 500 GB, while also improving storage and transfer efficiency._
```
Comment (davanstrien) [inline researcher-dataset-sharing.md:37]:
```suggestion
_Note: The [Xet team](https://huggingface.co/xet-team) is currently working on a backend update that will increase per-file limits from the current 50 GB to 500 GB while also improving storage and transfer efficiency._ 
```

**Review by davanstrien — APPROVED** [review]
Added a few nits but this is looking great. Thanks for working on this. Shall we aim to merge this week @cfahlgren1?

**Comment by cfahlgren1** [comment]
> Added a few nits but this is looking great. Thanks for working on this. Shall we aim to merge this week @cfahlgren1?

Yes, especially with the offsite. Plan for this to be one of my main focuses today!

File: researcher-dataset-sharing.md [hunk 11]
```diff
@@ -1,143 +1,205 @@
...
 thumbnail: /blog/assets/researcher-dataset-sharing/thumbnail.png
 authors:
 - user: davanstrien
+- user: cfahlgren1
+- user: lhoestq
+- user: erinys
 ---
 
-# Creating open machine learning datasets? Share them on the Hugging Face Hub!
+If you're working on data-intensive research or machine learning projects, you need a reliable way to share and host your datasets. Public datasets such as Common Crawl, MNIST, ImageNet, and more are critical to the open ML ecosystem, yet they can be challenging to host and share.
```
Comment (lhoestq) [inline researcher-dataset-sharing.md:11]:
```suggestion
If you're working on data-intensive research or machine learning projects, you need a reliable way to share and host your datasets. Public datasets such as Common Crawl, ImageNet, Common Voice and more are critical to the open ML ecosystem, yet they can be challenging to host and share.
```
MNIST is pretty small and uninteresting, I added Common Voice from mozilla which is a big speech dataset

File: researcher-dataset-sharing.md [hunk 12]
```diff
@@ -1,143 +1,205 @@
...
 
-## Who is this blog post for?
+Hugging Face Hub makes it seamless to host and share datasets, trusted by many leading research institutions, companies, and government agencies, including [Meta](https://huggingface.co/meta-llama), [Google](https://huggingface.co/google), [Stanford](https://huggingface.co/stanfordnlp), NASA(https://huggingface.co/ibm-nasa-geospatial) and [THUDM](https://huggingface.co/THUDM).
```
Comment (lhoestq) [inline researcher-dataset-sharing.md:13]:
```suggestion
Hugging Face Hub makes it seamless to host and share datasets, trusted by many leading research institutions, companies, and government agencies, including [Meta](https://huggingface.co/meta-llama), [Google](https://huggingface.co/google), [Stanford](https://huggingface.co/stanfordnlp), [NASA](https://huggingface.co/ibm-nasa-geospatial) and [THUDM](https://huggingface.co/THUDM).
```

File: researcher-dataset-sharing.md [hunk 13]
```diff
@@ -1,143 +1,205 @@
...
 
+## Third Party Library Support
 
-#### Datasets Viewer 
+Hugging Face is fortunate to have third party integrations with the leading open source data tools. By hosting a dataset on the Hub, it instantly makes the dataset compatible with the tools users are most familiar with.
 
-The datasets viewer allows people to explore and interact with datasets hosted on the Hub directly in the browser by visiting the dataset repository on the Hugging Face Hub. This makes it much easier for others to view and explore your data without first having to download it. The datasets viewer also allows you to search and filter datasets, which can be valuable to potential dataset users, understanding the nature of a dataset more quickly.
+Here are some of the libraries Hugging Face supports out of the box:
 
+| Library | Description | Monthly PyPi Downloads |
```
Comment (lhoestq) [inline researcher-dataset-sharing.md:84]:
```suggestion
| Library | Description | Monthly PyPi Downloads (2024) |
```

File: researcher-dataset-sharing.md [hunk 14]
```diff
@@ -1,143 +1,205 @@
...
+| [WebDataset](https://huggingface.co/docs/hub/datasets-webdataset) | Library to write I/O pipelines for large datasets. | **871K** |
+| [Argilla](https://huggingface.co/docs/hub/datasets-argilla) | Collaboration tool for AI engineers and domain experts that value high quality data. | **400k** |
 
-<p align="center"> 
- <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/researcher-dataset-sharing/datasets-viewer.png" alt="Screenshot of a dataset viewer on the Hub showing a named entity recognition dataset"><br> 
-<em>The dataset viewer for the multiconer_v2 Named Entity Recognition dataset.</em> 
- </p> 
+ Most of these libraries enable you to load or stream a dataset in 1 single line of code. 
+ 
+ Here's pandas as an example:
```
Comment (lhoestq) [inline researcher-dataset-sharing.md:96]:
```suggestion
 Here are some examples with Pandas, Polars and DuckDB:
```

File: researcher-dataset-sharing.md [hunk 15]
```diff
@@ -1,143 +1,205 @@
...
+## How can I host my dataset on the Hugging Face Hub? 
 
-Here are some resources to help you get started with sharing your datasets on the Hugging Face Hub:
+Now that you understand the benefits of hosting your dataset on the Hub, you might be wondering how to get started. Here are some comprehensive resources to guide you through the process:
 
 - General guidance on [creating](https://huggingface.co/docs/datasets/create_dataset) and [sharing datasets on the Hub](https://huggingface.co/docs/datasets/upload_dataset)
 - Guides for particular modalities:
   - Creating an [audio dataset](https://huggingface.co/docs/datasets/audio_dataset)
   - Creating an [image dataset](https://huggingface.co/docs/datasets/image_dataset)
+  - Creating a [video dataset](https://huggingface.co/docs/datasets/main/en/video_dataset)
```
Comment (lhoestq) [inline researcher-dataset-sharing.md:202]:
```suggestion
  - Creating a [video dataset](https://huggingface.co/docs/datasets/video_dataset)
```

File: researcher-dataset-sharing.md [hunk 16]
```diff
@@ -1,143 +1,205 @@
...
 
-#### Datasets Viewer 
+Hugging Face is fortunate to have third party integrations with the leading open source data tools. By hosting a dataset on the Hub, it instantly makes the dataset compatible with the tools users are most familiar with.
 
-The datasets viewer allows people to explore and interact with datasets hosted on the Hub directly in the browser by visiting the dataset repository on the Hugging Face Hub. This makes it much easier for others to view and explore your data without first having to download it. The datasets viewer also allows you to search and filter datasets, which can be valuable to potential dataset users, understanding the nature of a dataset more quickly.
+Here are some of the libraries Hugging Face supports out of the box:
 
+| Library | Description | Monthly PyPi Downloads |
+| :---- | :---- | :---- |
+| [Pandas](https://huggingface.co/docs/hub/datasets-pandas) | Python data analysis toolkit. | **258M** |
```
Comment (lhoestq) [inline researcher-dataset-sharing.md:86]: adding spark
```suggestion
| [Pandas](https://huggingface.co/docs/hub/datasets-pandas) | Python data analysis toolkit. | **258M** |
| [Spark](https://huggingface.co/docs/hub/datasets-spark) | 
Real-time, large-scale data processing tool in a distributed environment. | **29M** |
```

**Review by lhoestq — APPROVED** [review]
great !

File: researcher-dataset-sharing.md (hunk 3 above)
Comment (lhoestq) [reply researcher-dataset-sharing.md]: You could also add the DICE lab at NYU https://huggingface.co/nyu-dice-lab, and possibly the barcelona supercomputing center https://huggingface.co/BSC-LT

Note that nvidia also has important datasets on HF: https://huggingface.co/nvidia and could be cool to mention them
